apps/openmw/mwrender/sky.cpp

```diff
@@ -1422,7 +1432,7 @@ class WrapAroundOperator : public osgParticle::Operator
         osg::Matrix toWorld, toLocal;
 
         std::vector<osg::Matrix> worldMatrices = ps->getWorldMatrices();
- 
```

Comment:
unnecessary formatting change

Reply:
yes

Reply:
in fact i  removed an unecessary whitespace